### DISCUSSION TITLE
Column edit - Illegal character

### DISCUSSION THREAD

**Discussion by Overness** [Budibase/budibase#8664] [Help] [answered]
Hello,

Using underscores being quite standard in column names, is there a plan to allow their use?
<img width="391" alt="Screenshot 2022-11-12 at 7 31 18 PM" src="https://user-images.githubusercontent.com/52577546/201477938-c8c00276-cbe9-489d-8d91-a9ee923a8b58.png">

**Comment by melohagan** [comment, marked as answer]
Hey @Overness 

Sorry, I had accidentally not included underscores in the regex. There's a PR out now: https://github.com/Budibase/budibase/pull/8694
